_episodes/03-merge_conflicts.md

```diff
@@ -29,15 +29,15 @@ git switch main
 git stage ingredients.md
 git commit -m "Reduce salt"
 git switch experiment
-# change line to 3 tsp in ingredients.md
+# change line to 3 tsp salt in ingredients.md
 git stage ingredients.md
 git commit -m "Added salt to balance coriander"
 git graph
 ```
 {: .commands}
 ```
 * d5fb141 (HEAD -> experiment) Added salt to balance coriander
-| * 7477632 (main) reduce salt
+| * 7477632 (main) Reduce salt
```

Comment:
This isn't particularly important, but if the commit message has changed, won't that mean the hash should have changed too?

Reply:
Yes, you are right. This one I edited because in the steps above, the commit message was "Reduce salt" with a capital R. So I made the edit to get a consistent output. This was a minor edit, so I didn't want to change the entire commit history that follows. Hope this is ok.

Reply:
Totally fine!